src/extension.ts

```diff
@@ -33,6 +33,8 @@ const connectedModeServersSectionName = 'connectedMode.servers';
 const connectedModeProjectSectionName = 'connectedMode.project';
 
 function runJavaServer(context: VSCode.ExtensionContext): Thenable<StreamInfo> {
+  const enableDebug = getSonarLintConfiguration().get('debug')
```

Comment:
This will only be evaluated at startup right? Would be better to support a dynamic change of this property, because sometimes restarting the IDE make it difficult to reproduce an issue.